src/course-unit/xblock-container-iframe/messages.ts

```diff
@@ -0,0 +1,10 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  xblockIframeTitle: {
+    id: 'course-authoring.course-unit.xblock.iframe.title',
+    defaultMessage: 'Course unit iframe',
```

Comment:
[nit] Probably, we should add `description` to all new messages to stick to a common approach.

Reply:
In this repository we don't have a single generally accepted rule for adding descriptions for translations. I suggest leaving it as is. Thanks for paying attention! 💯

Reply:
In the past we weren't consistent but the new policy is that all new messages must have descriptions.

Reply:
Thanks! Added description in [`b7b5d68` (#1375)](https://github.com/openedx/frontend-app-authoring/pull/1375/commits/b7b5d681ee1faa1a836b59a87f82b5aabb459e3a)